engine/apps/api/serializers/schedule_base.py

```diff
@@ -68,7 +68,7 @@ def get_warnings(self, obj):
 
     def get_on_call_now(self, obj):
         # Serializer context is set here: apps.api.views.schedule.ScheduleView.get_serializer_context
-        users = self.context["oncall_users"].get(obj.pk, [])
+        users = self.context["oncall_users"].get(obj, [])
```

Comment:
`oncall_users` set on this context object now has `User` objects as keys rather than the string pks